src/inference/include/openvino/runtime/core.hpp

```diff
@@ -148,7 +142,11 @@ class OPENVINO_RUNTIME_API Core {
               class... Properties,
               std::enable_if_t<std::is_same_v<Path, std::filesystem::path> && (sizeof...(Properties) > 0)>* = nullptr>
     auto read_model(const Path& model_path, const Path& bin_path, Properties&&... properties) const {
```

Comment:
Please check with original implementation there were some default parameters which should be restored, The API should not be changed.

Reply:
Fixed in right place.